packages/unpublished-debug-encapsulation-test-app/testem.js

```diff
@@ -2,17 +2,34 @@
 const customDotReporter = require('@ember-data/unpublished-test-infra/src/testem/custom-dot-reporter');
 
 const TestIE = process.env.TEST_IE11;
+const TestSafari = process.env.TEST_SAFARI;
 
 if (TestIE) {
   // eslint-disable-next-line no-console
   console.log('\n\nLaunching with IE\n\n');
 }
+if (TestSafari) {
+  // eslint-disable-next-line no-console
+  console.log('\n\nLaunching with Safari\n\n');
+}
+
+function browserPicker() {
+  if (TestSafari) {
+    return ['Safari'];
+  }
+
+  if (TestIE) {
+    return ['IE'];
+  }
+
+  return ['Chrome'];
+}
```

Comment:
If I got this right, perhaps we could move `browserPicker` to `unpublished-test-infra`